crates/oxc_linter/src/context.rs

```diff
@@ -42,11 +42,11 @@ impl<'a> LintContext<'a> {
         self.diagnostics.borrow_mut().push(diagnostic.into());
     }
 
-    pub fn fix(&self, fix: Fix) {
+    pub fn fix(&self, fix: Fix<'a>) {
         self.fixes.borrow_mut().push(fix);
     }
 
-    pub fn into_fixes(self) -> Vec<Fix> {
+    pub fn into_fixes(self) -> Vec<Fix<'a>> {
```

Comment:
Is this used anymore? I think I added it and realised `into_diagnostics` takes ownership of self, so it wouldn't be that useful

Reply:
There is no use of this method, should we delete it?

Reply:
🗑️ it